specs/babbage/plutus.cddl

```diff
@@ -1,3 +1,15 @@
+; bounded_bytes = bytes .size (0..64)
```

Comment:
Wouldn't this affect CIP25 as well if the CDDL isn't accurate?

Reply:
CIP25 is a subset of the metadata. The way this is defined here is basically the same as how CBOR **indefinite** byte strings are encoded, but with the restriction that *each definite segment* is at most 64-bytes long, whereas how cip25 is defined the bytes there can be encoded indefinite but with *the sum of all definite segments* being 64-bytes max, thus still abiding by this true bounded_bytes definition without needing to specify it.

Reply:
actually that's not even an issue with cip25 since bounded bytes it only used in plutus datums, not metadatums so the two things never even touch each other